sdk/formrecognizer/azure-ai-formrecognizer/dev_requirements.txt

```diff
@@ -3,4 +3,4 @@
 -e ../../cognitiveservices/azure-mgmt-cognitiveservices
 ../../core/azure-core
 -e ../../identity/azure-identity
-aiohttp>=3.0; python_version >= '3.6'
\ No newline at end of file
+aiohttp>=3.0; python_version >= '3.7'
```

Comment:
Now that I look at this again, I think we can remove the conditional, but keep the >=3.0

Reply:
I just checked with @scbedd and this should be fine, good catch

Reply:
Though it's an unecessary test, it's not having an bad impact, so not critical

Reply:
Ok will leave as is for now. Thanks for the reviews!